### DISCUSSION TITLE
Tauri 2.0 mobile - Is it like RN

### DISCUSSION THREAD

**Discussion by Fnux8890** [tauri-apps/tauri#9958] [General]
I recently saw the announcement of tauri 2.0 and also the tauri for mobile announcements (a little behind I know 😅)

Im just curious about if Tauri aims to be like RN in the sense that when mobile applications are developed its actual native components that are used or is there an abstraction between like with NativeScript?

In a somewhat biased opinion the RN approach seems to be the better one for delivering cross platform experiences that abide to what the native systems deliver and how it acts.
This could be awesome for Rust. But as of what I can see it seems like it's more of a NativeScript Situation with a browser view in-between. Not that there anything wrong with that I'm just curious.

**Comment by FabianLars** [comment]
idk exactly how nativescript works but i assume it's similar to capacitor/ionic which is essentially the same that tauri is doing. I think the react native approach is very unrealistic for frontend-framework independent projects like those.
